pages/docs/integrations/grouparoo-onesignal.mdx

```diff
@@ -0,0 +1,146 @@
+  alt="OneSignal Setup Guidance"
+  width={733}
+  height={98}
+/>
+
+<br />
+
+## App
+
+To get ready to sync from Grouparoo to OneSignal, you [create an App](https://www.grouparoo.com/docs/config/apps) which tells Grouparoo the details of how to connect to OneSignal. This specifies the account used for any Destinations.
+
+### App Options
+
+In Grouparoo’s UI config, you specify the options to connect to OneSignal and test the connection. Then Update your configuration to save it.
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/onesignal/app/options.png"
+  alt="OneSignal App Options"
+  width={733}
+  height={259}
+/>
+
+<br />
+<br />
+
+<table className="docTable">
+  <tablebody>
+    <tr>
+      <td>Option</td>
+      <td>Description</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>App ID</strong> (required)
+      </td>
+      <td>OneSignal App ID. Found in the app’s Settings > Keys & IDs.</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>API Key</strong> (required)
+      </td>
+      <td>OneSignal REST API Key. Found in the app’s Settings > Keys & IDs</td>
+    </tr>
+  </tablebody>
+</table>
+
+## OneSignal Export Users
+
+This Destination allows you to sync your Grouparoo Records to Users in OneSignal.
+
+Companies use OneSignal to organize their information about customers in order to facilitate communications over multiple web and mobile platforms.
+
+When you [create a Destination](https://www.grouparoo.com/docs/config/destinations), you make choices about what data to sync.
+
+### Sync Modes
+
+A Destination’s sync mode determines what to update in a Destination. OneSignal supports only enrich mode:
+
+- **Enrich**: Only update those Users that already exist in OneSignal. Do not add or remove Users. This option will only update existing Users in OneSignal.
+
+### Destination Options
+
+Some types of Destinations require further clarification about how data should sync. In the case of OneSignal, there are no further Options to configure before moving on to mapping Record Data.
+
+### Record Data
+
+Mapping Grouparoo Record Properties to OneSignal Tags is the key to syncing data to OneSignal Users.
+
+OneSignal can be used for customizing marketing campaigns and customizing the content to the audience. They allow you to personalize communications and segment your audience into lists for targeted mailings, push notifications, and SMS messages.
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/onesignal/export-users/properties.png"
+  alt="OneSignal Export Users Properties"
+  width={733}
+  height={550}
+/>
+
+<br />
+<br />
+
+#### Required Properties
+
+You must map a Grouparoo Property to the **external_id** OneSignal Tag. Choose the Grouparoo Property that uniquely identifies the OneSignal User from the dropdown menu.
+
+- **external_id:** A User’s unique identifier that relates the Source Record and the Grouparoo Property to the User in OneSignal.
```

Comment:
The description here mentions `external_id`, but I see `external_user_id` in the screenshots. Could we confirm which one it is and make the changes as necessary?

Reply:
good catch. external_user_id is the correct one.